xml/storage_multipath.xml

```diff
@@ -447,6 +390,23 @@ To disable multipath support in <filename>initrd</filename>, run the command:
      the changes may require rescanning SCSI targets and reconfiguring
      multipathing on the host.
    </para>
+   <para>
+    If a disk array has more than one storage processor, ensure that the SAN
```

Comment:
Are there any arrays where there are more than 2 SP? I got confused by the "both storage processor" mentioned later. Usually there are 2 SP if I understand it correctly?

Reply:
I can change "both" to "all" if you prefer.

Reply:
I'd delete this paragraph. IMHO it does not convey valuable information. Instead there could be a small paragraph about the purpose of storage processors?

Reply:
Have you seen the version of this section in my last PR? I believe it's better understandable now.

Reply:
Yes, that looks good to me.